platforms/chibios/drivers/vendor/RP/RP2040/ws2812_vendor.c

```diff
@@ -256,7 +259,7 @@ static inline void sync_ws2812_transfer(void) {
     }
 
     // Busy wait until last transfer has finished
-    while (unlikely(!timer_expired32(chSysGetRealtimeCounterX(), LAST_TRANSFER))) {
+    while (unlikely(!time_reached(LAST_TRANSFER))) {
     }
```

Comment:
```suggestion
    busy_wait_until(LAST_TRANSFER);
```
(`busy_wait_until()` is in `hardware_timer` too)

Reply:
Done.